config.py

```diff
@@ -1,3 +1,4 @@
 ACCESS_TOKEN = '<ACCESS_TOKEN>'
 VERIFY_TOKEN = '<VERIFY_TOKEN>'
 WIT_AI_ACCESS_TOKEN = 'IKJJJYYVR3X672DHFVS7U7C4L2MQSS2P'
+OPEN_WEATHER_MAP_ACCESS_TOKEN = 'adee94563461cdf1dcfe25956aea10ae'
```

Comment:
Replace the token with the placeholder '<OPEN_WEATHER_MAP_ACCESS_TOKEN>', i've set it up on the server.